DevelopedCalculator.js

```diff
@@ -19,8 +19,6 @@ class DevelopedCalculator extends Calculator {
     onButtonNumberClick(event) {
```

Comment:
is this method needed?
you declared this method to push current result to the history array 
but now you removed that code, so do you still need this method?